week6_outro/seq2seq/practice_tf.ipynb

```diff
@@ -7,8 +7,9 @@
    "outputs": [],
    "source": [
     "import sys, os\n",
-    "if 'google.colab' in sys.modules and not os.path.exists('.setup_complete'):\n",
-    "    %tensorflow_version 1.x\n",
+    "if 'google.colab' in sys.modules:\n",
+    "    # https://github.com/yandexdataschool/Practical_RL/issues/256\n",
+    "    !pip install tensorflow-gpu==1.13.1\n",
```

Comment:
This is incorrect b/c it will reinstall `tensorflow-gpu` each time this cell is executed. The `if not os.path.exists('.setup_complete')` clause should be at the top.